cms/envs/aws.py

```diff
@@ -429,6 +433,11 @@
 
 VIRTUAL_UNIVERSITIES = ENV_TOKENS.get('VIRTUAL_UNIVERSITIES', [])
 
+# Zendesk
+# ZENDESK_USER = AUTH_TOKENS.get("ZENDESK_USER")
+# ZENDESK_API_KEY = AUTH_TOKENS.get("ZENDESK_API_KEY")
```

Comment:
Will we need these? I don't see them used in the React contact us form in the LMS?

Reply:
I think you're right that we don't need these

Reply:
This probably depends on the API requirements, without knowing those atm I'd say we can leave it here for now as is.  Then we can undocument it if we realize it's information that we need when we integrate the actual API request in SFE.